src/connection/ConnectionOptionsReader.ts

```diff
@@ -11,6 +11,11 @@ import {ConnectionOptionsXmlReader} from "./options-reader/ConnectionOptionsXmlR
  */
 export class ConnectionOptionsReader {
 
+    static findOptionByName(options: ConnectionOptions[], name = "default") {
```

Comment:
Can we keep the API for `ConnectionOptionsReader` leaner?  I don't think we need to add public helper methods onto the API.

Reply:
yep, need will create a pure function